languages/java/exercises/concept/switch-statement/.docs/after.md

```diff
@@ -0,0 +1,134 @@
+Starting with Java 14 (available as a preview before in Java 12 and 13) it is possible to use the "enhanced" switch implementation.
+
+1. You have the possiblity to assign multiple value in a sigle case.
+   Instead of :
+
+   ```java
+   switch (number) {
+       case 1:
+           //do stuff
+           break;
+       case 2:
+           //do different stuff
+           break;
+       case 3:
+           //do same stuff as case 1
+           break;
+       (...)
+   }
+   ```
+
+   You can do :
+
+   ```java
+   switch (number) {
+       case 1, 3:
+           //do stuff
+           break;
+       case 2:
+           //do other stuff
+           break;
+       (...)
+   }
+   ```
+
+2. The `switch-statement` is now a `switch expression`. What is the difference ?
+   Basicly a statement is expecting some strict logic where an expression can return a value.
+   Instead of :
```

Comment:
Is "The `switch-statement` is now a `switch expression`" correct? Reading the docs, it seems to me like one writes either a switch statement or a switch expression.

```suggestion
2. The `switch-statement` is now a `switch expression`. What is the difference ?
   Basicly a statement is expecting some strict logic where an expression can return a value.
   Instead of :
```